nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -53,6 +85,7 @@ def _get_node_type_and_metatype(node: torch.fx.Node) -> Tuple[str, om.OperatorMe
                 # TODO(dlyakhov): get correct nodes types from this nodes as well
                 node_type = str(node.target)
             node_metatype = PT_OPERATOR_METATYPES.get_operator_metatype_by_op_name(node_type)
+            node_metatype = GraphConverter._get_node_subtype(node, node_metatype, model)
```

Comment:
Please dont create alternative mechanism to determinate subtypes.
If you want to use PTMetatype you should integrate it to existed `PTMetatype.determine_subtype` method.

Reply:
Done, please check